src/vs/workbench/services/tunnel/electron-sandbox/tunnelService.ts

```diff
@@ -73,8 +75,13 @@ export class TunnelService extends AbstractTunnelService {
 		});
 	}
 
+	private get defaultTunnelHost(): string {
```

Comment:
This is now duplicated from `BaseTunnelService`. We should move it into a shared location like `AbstractTunnelService`.

Reply:
done